File: 231A - Team.py
def aTeam(inp):
  count = 0
  for i in range(inp):
    prob = input().split(' ')
    for j in range(len(prob)):
      prob[j] = int(prob[j])
    if sum(prob) >= 2:
      count+=1
  print(count)
aTeam(int(input()))

# Alternative -> Better Solution

def aTeam(inp):
    count = 0
    for _ in range(inp):
        prob = list(map(int, input().split()))
        if sum(prob) >= 2:
            count += 1
    print(count)

aTeam(int(input()))

File: 231A - Team.js
function aTeam(inp) {
  var count = 0;
  for (i = 0; i < inp; i++) {
    var prob = readline().split(" ");

    var sum = prob.reduce((accumulator, currentValue) => {
      return accumulator + parseInt(currentValue);
    }, 0);
    if (sum >= 2) {
      count += 1;
    }
  }

  print(count);
}

aTeam(parseInt(readline()));
